Lykke.Service.LykkeService/src/Lykke.Service.LykkeService/Program.cs

```diff
@@ -1,56 +1,17 @@
-﻿using System;
-using System.IO;
+﻿using Lykke.Sdk;
 using System.Threading.Tasks;
-using Lykke.Sdk;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.PlatformAbstractions;
 
 namespace Lykke.Service.LykkeService
 {
     internal sealed class Program
     {
-        public static string EnvInfo => Environment.GetEnvironmentVariable("ENV_INFO");
-
         public static async Task Main(string[] args)
         {
-            Console.WriteLine($"{PlatformServices.Default.Application.ApplicationName} version {PlatformServices.Default.Application.ApplicationVersion}");
 //#$if DEBUG
```

Comment:
remove these // ? Now it will first start the debug version and then the release version

Reply:
it's required so during solution creation from template such compiler directives won't vanish